docs/reference/equality.md

```diff
@@ -17,13 +17,10 @@ In Kotlin there are two types of equality:
 Referential equality is checked by the `===` operation (and its negated counterpart `!==`). `a === b` evaluates to
 true if and only if `a` and `b` point to the same object.
 
-Alternatively, you can check for reference equality using a built-in inline function `identityEquals()`
-that can be called in the following way
+Alternatively, you can check for reference equality using a built-in inline function `identityEquals()`:
```

Comment:
I believe we should get rid of `identityEquals` in favor of `===`, but this is subject to discussion